openapic/compiler.go

```diff
@@ -33,14 +34,33 @@ import (
 )
 
 type PluginCall struct {
-	Name   string
-	Output string
+	Name       string
+	Invocation string
 }
 
 func (pluginCall *PluginCall) perform(document *openapi_v2.Document, sourceName string) {
 	if pluginCall.Name != "" {
 		request := &plugins.PluginRequest{}
-		request.Parameter = ""
+
+		request.Invocation = pluginCall.Invocation
+		invocationParts := strings.Split(pluginCall.Invocation, ":")
+		var outputLocation string
+		switch len(invocationParts) {
+		case 1:
+			outputLocation = invocationParts[0]
+		case 2:
+			parameters := strings.Split(invocationParts[0], ",")
```

Comment:
Consider adding regex validation for the syntax. it might be hard to people to get it right when trying this for the first time.